aspnetcore/fundamentals/dependency-injection.md

```diff
@@ -109,40 +91,27 @@ In dependency injection terminology, a service:
 
 The framework provides a robust [logging](xref:fundamentals/logging/index) system. The `IMyDependency` implementations shown in the preceding examples were written to demonstrate basic DI, not to implement logging. Most apps shouldn't need to write loggers. The following code demonstrates using the default logging, which doesn't require any services to be registered in `ConfigureServices`:
 
-[!code-csharp[](dependency-injection/samples/3.x/DependencyInjectionSample/Pages/About.cshtml.cs?name=snippet)]
+[!code-csharp[](dependency-injection/samples/6.x/DependencyInjectionSample/Pages/About.cshtml.cs?name=snippet)]
 
 Using the preceding code, there is no need to update `ConfigureServices`, because [logging](xref:fundamentals/logging/index) is provided by the framework.
 
-## Services injected into Startup
-
-Services can be injected into the `Startup` constructor and the `Startup.Configure` method.
-
-Only the following services can be injected into the `Startup` constructor when using the Generic Host (<xref:Microsoft.Extensions.Hosting.IHostBuilder>):
+## Services injected into Program.c
 
-* <xref:Microsoft.AspNetCore.Hosting.IWebHostEnvironment>
-* <xref:Microsoft.Extensions.Hosting.IHostEnvironment>
-* <xref:Microsoft.Extensions.Configuration.IConfiguration>
-
-Any service registered with the DI container can be injected into the `Startup.Configure` method:
+Services can be injected into the *Program.c* file.
```

Comment:
@halter73 What's the correct wording here now that top level statements are in effect?
https://docs.microsoft.com/en-us/dotnet/csharp/fundamentals/program-structure/top-level-statements

Reply:
@halter73 this was an easy migration to minimal host model and everything seems to work. Is there anything with DI that would change other than what I pointed out?